hr_employee_relative/migrations/12.0.1.0.0/post-migration.py

```diff
@@ -0,0 +1,50 @@
+# Copyright 2019 Eficent <http://www.eficent.com>
+# Copyright 2019 Tecnativa - Pedro M. Baeza
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+from openupgradelib import openupgrade
+
+
+@openupgrade.migrate()
+def migrate(env, version):
+    cr = env.cr
+    columns = 'fam_spouse, fam_spouse_employer, fam_spouse_tel, fam_father,' \
+              ' fam_father_date_of_birth, fam_mother, fam_mother_date_of_birth'
+    cr.execute('SELECT id, %s FROM hr_employee' % columns)
+
+    relation_spouse = env.ref('hr_employee_relative.relation_spouse').id
+    relation_parent = env.ref('hr_employee_relative.relation_parent').id
+    relation_child = env.ref('hr_employee_relative.relation_child').id
+
+    for employee in cr.fetchall():
+        if employee[1] or employee[2] or employee[3]:
+            env['hr.employee.relative'].create({
+                'employee_id': employee[0],
+                'name': employee[1] or 'Spouse',
```

Comment:
What I don't understand: why should be always '`Spouse`' and never a '`Husband`'? 😏

Reply:
In hr_family the field was named `Spouse`. It works for both genders